test/integration/bad/function-signatures/overloading/stanc.expected

```diff
@@ -0,0 +1,219 @@
+  $ ../../../../../../install/default/bin/stanc no_minimum_dae.stan
+Semantic error in 'no_minimum_dae.stan', line 38, column 12 to column 59:
+   -------------------------------------------------
+    36:    array[4] vector[3] y_hat;
+    37:    {
+    38:      y_hat = dae(chem_dae, yy0, yp0, t0, ts, theta, x, 0, 0);
+                     ^
+    39:    }
+    40:  
+   -------------------------------------------------
+
+No unique minimum promotion found for function "chem_dae".
+Overloaded functions must not have multiple equally valid promotion paths.
+This function has several:
+(real, vector, vector, array[] real, array[] real, real, int) => vector
+(real, vector, vector, array[] real, array[] real, int, real) => vector
```

Comment:
Could we have the error also output what the user passed in? Like it might be nice for the error to be 
```suggestion
No unique minimum promotion found for function "chem_dae" with argument signature:
(real, vector, vector, array[] real, array[] real, int, int) => vector
Overloaded functions must not have multiple equally valid promotion paths.
This function has several:
(real, vector, vector, array[] real, array[] real, real, int) => vector
(real, vector, vector, array[] real, array[] real, int, real) => vector
```

Extremely bonus points / not necessary, but would it be possible to show in bold which arguments we had trouble detecting promotion for?

Reply:
I don't think there is a way to do bold which works both in the console and in raw text output (e.g. in stancjs)